honeybee_energy/load/lighting.py

```diff
@@ -46,9 +46,10 @@ class Lighting(_LoadBase):
         * visible_fraction
         * convected_fraction
         * baseline_watts_per_area
+        * user_data
     """
     __slots__ = ('_watts_per_area', '_schedule', '_return_air_fraction',
-                 '_radiant_fraction', '_visible_fraction', '_baseline_watts_per_area')
+                 '_radiant_fraction', '_visible_fraction', '_baseline_watts_per_area', '_user_data')
```

Comment:
Remove from `__slots__` since it's already in the parent class.

Reply:
commit: loads cleaned up